gearbot_2/src/cache/mod.rs

```diff
@@ -26,7 +26,8 @@ pub struct Cache {
 }
 
 impl Cache {
-    pub fn new() -> Self {
+    #[allow(clippy::new_without_default)]
```

Comment:
is there a reason we don't just derive default and use that, deleting this `new` constructor (or implementing it with default if you like the `new` wording?).

Reply:
the idea was more semantically that i'm not sure there should be a "default" cache as there is nothing else

Reply:
I see. I guess that makes enough sense then.